components/staticFeeds/sample-messages.json

```diff
@@ -118,6 +118,199 @@
+      "titleShort": "No group. No date. High priority.",
+      "description": "No group. No date. High priority.",
+      "descriptionShort": "No group. No date. High priority.",
+      "messageType": "notification",
+      "goLiveDate": null,
+      "expireDate": null,
+      "featureImageUrl": null,
+      "priority": "high",
+      "audienceFilter": {
+      "groups": [],
+      "dataUrl": null,
+      "dataObject": null,
+      "dataArrayFilter": null
+      },
+      "actionButton": {
+      "label": "Go",
+      "url": "http://www.google.com"
+      },
+      "moreInfoButton": null,
+      "confirmButton": null
+    },
+    {
+      "id": "demo-low-priority-no-group-no-date",
+      "title": "No group. No date. Low priority.",
+      "titleShort": "No group. No date. Low priority.",
+      "description": "Test no group no date. Low priority.",
+      "descriptionShort": "No group. No date. Low priority.",
+      "messageType": "notification",
+      "goLiveDate": null,
+      "expireDate": null,
+      "featureImageUrl": null,
+      "priority": null,
+      "audienceFilter": {
+      "groups": [],
+      "dataUrl": null,
+      "dataObject": null,
+      "dataArrayFilter": null
+      },
+      "actionButton": {
+      "label": "Go",
+      "url": "http://www.google.com"
+      },
+      "moreInfoButton": null,
+      "confirmButton": null
+    },
+    {
+      "id": "demo-low-priority-valid-group-no-date",
+      "title": "Valid group. No date. Low.",
+      "titleShort": "Valid group. No date. Low priority.",
+      "description": "Valid group. No date. Low priority.",
+      "descriptionShort": "Valid group. No date. Low priority.",
+      "messageType": "notification",
+      "goLiveDate": null,
+      "expireDate": null,
+      "featureImageUrl": null,
+      "priority": null,
+      "audienceFilter": {
+      "groups": ["Portal Administrators"],
+      "dataUrl": null,
+      "dataObject": null,
+      "dataArrayFilter": null
+      },
+      "actionButton": {
+      "label": "Go",
+      "url": "http://www.google.com"
+      },
+      "moreInfoButton": null,
+      "confirmButton": null
+    },
+    {
+      "id": "demo-high-priority-valid-group-no-date",
+      "title": "Valid group. No date. High priority.",
+      "titleShort": "Valid group. No date. High priority.",
+      "description": "Valid group. No date. High priority.",
+      "descriptionShort": "Valid group. No date. High priority.",
+      "messageType": "notification",
+      "goLiveDate": null,
+      "expireDate": null,
+      "featureImageUrl": null,
+      "priority": "high",
+      "audienceFilter": {
+      "groups": ["Portal Administrators"],
+      "dataUrl": null,
+      "dataObject": null,
+      "dataArrayFilter": null
+      },
+      "actionButton": {
+      "label": "Go",
+      "url": "http://www.google.com"
+      },
+      "moreInfoButton": null,
+      "confirmButton": null
+    },
+    {
+      "id": "demo-low-priority-valid-group-valid-date",
```

Comment:
Is the most salient characteristic of the `date` that it's `valid`? Or that it's `current`, as in the current date is between the beginning and end dates? Non-current dates wouldn't be "invalid", they'd be valid expressions of old dates?